app/models/queue_item.rb

```diff
@@ -0,0 +1,23 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  def video_title
+    video.title
+  end
+
+  def rating 
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
You should use `find_by` for single record search, rather than inefficient `where(...).first`

`review = Review.find_by(user_id: user.id, video_id: video.id)`